packages/demo-app/src/styles/_examples.scss

```diff
@@ -0,0 +1,146 @@
+@import "~@blueprintjs/colors/lib/scss/colors";
+@import "~@blueprintjs/core/lib/scss/variables";
+
+$background-color: $light-gray2;
+$content-background-color: $light-gray4;
+$dark-background-color: $dark-gray2;
+$dark-content-background-color: $dark-gray3;
+
+$intent-primary-text: $blue1;
+$intent-success-text: $green1;
+$intent-warning-text: $orange1;
+$intent-danger-text: $red1;
+
+$dark-intent-primary-text: $blue5;
+$dark-intent-success-text: $green5;
+$dark-intent-warning-text: $orange5;
+$dark-intent-danger-text: $red5;
+
+.examples-root {
+  align-items: center;
+  background-color: $background-color;
+  display: flex;
+  flex-direction: row;
+}
+
+.examples-container {
+  background-color: $background-color;
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  width: 100%;
+
+  &.#{$ns}-dark {
+    background-color: $dark-background-color;
+  }
+}
+
+.example-row {
+  display: flex;
+  justify-content: space-evenly;
+
+  > :not(:last-child) {
+    margin-right: 10px;
+  }
+}
+
+.example-card-container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 30px;
+}
+
+.example-card {
+  background-color: $content-background-color;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
+
+  > :not(:last-child) {
+    margin-bottom: 10px;
+  }
+
+  &.horizontal {
+    flex-direction: row;
+    justify-content: center;
+
+    > :not(:last-child) {
+      margin-bottom: 0;
+      margin-right: 10px;
+    }
+  }
+}
+
+.tag-container {
+  display: flex;
+  flex-direction: column;
+
+  > :not(:last-child) {
+    margin-bottom: 10px;
+  }
+}
+
+.html-table-example {
+  .#{$ns}-intent-primary {
```

Comment:
this was a little surprising to me. it's fine for now, but I'm surprised we don't already have a public API to do this without applying custom CSS. something like `<Text intent="...">`

Reply:
me too. maybe this can be added in the 5.x release - shouldn't be breaking, right?

Reply:
Just filed https://github.com/palantir/blueprint/issues/4930. it doesn't need to be tied to 5.0, since it's non-breaking